codelabs/gke-base/install/setup.sh

```diff
@@ -15,8 +15,29 @@ if [ -z "$PROJECT_ID" ]; then
   exit 1
 fi
 
+if [ -z "$1" ]; then
+  err "No codelab specified as the first argument to this script. Exiting."
+  exit 1
+fi
+
+gsutil cp gs://gke-spinnaker-codelab/$1/overrides.tar .
```

Comment:
Are you planning on creating documentation for how to leverage this script in new codelabs beyond usage examples?

Reply:
yes